src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Figure implements Drawable {
+    private String figureName;
+    private double figureArea;
```

Comment:
I suggest to remove this field and calculate area each time we want it, in this case we will not store unnecessary information in objects